### PR TITLE
Add Amy Repo Keyring

### PR DESCRIPTION
**Pull request by elihwyma**
Adds keys for https://beta.anamy.gay

### CODE REVIEW

File: makefiles/amy-keyring.mk [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	# amy-keyring.mk Prep amy-keyring
+	cp -a $(BUILD_STAGE)/amy-keyring $(BUILD_DIST)
+
+	# amy-keyring.mk Make .debs
+	$(call PACK,amy-keyring,DEB_AMY_KEYRING_V)
+
+	# amy-keyring.mk Build cleanup
+	rm -rf $(BUILD_DIST)/amy-keyring
+
+.PHONY: amy-keyring amy-keyring-package
```
Comment (TheRealKeto) [inline makefiles/amy-keyring.mk:32]: Add an extra line to your Makefile at the end.

**Review by TheRealKeto — CHANGES_REQUESTED** [review]
This should be merged soon, but fix these mistakes first. Additionally, I might push for keyrings (and packages dealing with them) to have a better approach (e.g the keyring should come from an external source just like a tarball), but that will changed later.

**Comment by sunflsks** [comment]
As soon as what keto pointed out is fixed, I can merge this

**Review by TheRealKeto — APPROVED** [review]

**Review by m1stadev — APPROVED** [review]
LGTM :)

**Pull request merged**